sdc/crypto/key_store.py

```diff
@@ -59,13 +66,22 @@ def get_key_by_kid(self, purpose, kid, key_type):
         else:
             return key
 
-    def get_key_for_purpose_and_type(self, purpose, key_type):
+    def get_key(self, *, purpose, key_type, service=None):
```

Comment:
For impacted services reviewers; While this change is being introduced as a non-breaking API change to the libraries intended usage, if this (publically available) internal method is called outside of that intended usage it will need to be dealt with as a breaking change.